changes/changed/fix-missing-dereg-event.md

```diff
@@ -0,0 +1,8 @@
+#cnight-generates-dust
+#runtime
+# Fix missing `Deregistration` event when user adds second mapping
+
+When a user adds a second mapping on Cardano, this invalidates their existing registration. Therefore, we should emit a deregistration event.
```

Comment:
Is there a more fundamental issue with language in the cardano registration here? Should they be "setting the mapping" rather than "adding a mapping"?